### PR TITLE
Version

### LINKED ISSUE #9: Setup.py has no version
In the `setup.py` the version needs to be set hard https://github.com/KiraPC/fastapi-router-controller/blob/31fa13b8031a21054509e7deee7f4f76a1d9d3a0/setup.py#L17 won't work. pip or setuptools are also checking locally the version,  but this obviously won't work as env variable `RELEASE_VERSION` is not set (and also should not be). A simple `sed` and writing the number is sufficient.

### PR DESCRIPTION
**Pull request by delijati**
Set version

### CODE REVIEW

File: .github/workflows/on_tag_release.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Create Distribution Package
         env:
           RELEASE_VERSION: ${{ steps.vars.outputs.tag }}
-        run: python setup.py sdist
+        run: sed -i -r s"/VERSION = \"(.*)\"/VERSION = \"$FOO\"/"g setup.py && python setup.py sdist
```
Comment (KiraPC) [inline .github/workflows/on_tag_release.yml:31]: @delijati yes, ok. Could you change FOO with the correct value?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: fastapi-router-controller
```
Comment (KiraPC) [inline .github/workflows/ci.yml:1]: @delijati Could you remove and use the one already present?

File: tests/test_cbv.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import unittest
```
Comment (KiraPC) [inline tests/test_cbv.py:1]: @delijati Could you remove in favor to example?

**Review by KiraPC — APPROVED** [review]

**Comment by delijati** [comment]
awesome

**Pull request merged**